src/string/camelCase.js

```diff
@@ -6,11 +6,25 @@ define(['../lang/toString', './replaceAccents', './removeNonWord', './upperCase'
         str = toString(str);
         str = replaceAccents(str);
         str = removeNonWord(str)
-            .replace(/[\-_]/g, ' ') //convert all hyphens and underscores to spaces
-            .replace(/\s[a-z]/g, upperCase) //convert first char of each word to UPPERCASE
-            .replace(/\s+/g, '') //remove spaces
-            .replace(/^[A-Z]/g, lowerCase); //convert first char to lowercase
-        return str;
+            .replace(/[\-_]/g, ' '); // convert all hyphens and underscores to spaces
```

Comment:
I think we should remove the comments after! What do you guys think?

More readability?